### PR TITLE
BZ119170: JCA Resource Adapter creation error is 'Unknown Error'

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

**Comment by heiko-braun** [comment]
We need this in master first. Otherwise the change will be lost in the next version. Can submit another PR, add iding a note that this needs to be backported to 2.5.x? We'll do the backlport then.

File: gui/src/main/java/org/jboss/as/console/client/shared/subsys/jca/ResourceAdapterPresenter.java [hunk 1]
```diff
@@ -364,12 +367,15 @@ public void onCreateAdapter(final ResourceAdapter ra) {
...
-            @Override
-            public void onFailure(Throwable caught) {
-                super.onFailure(caught);
-                loadAdapter(false);
-            }
+        	
+			@Override
+			public void onFailure(Throwable caught) {
+				Log.error("Unknown error", caught);
+				Console.getMessageCenter().notify(
```
Comment (heiko-braun) [inline gui/src/main/java/org/jboss/as/console/client/shared/subsys/jca/ResourceAdapterPresenter.java]: Please use one of Console.error/info/debug instead. Using the message center directly is not recommened and could be removed in the future

**Comment by rpelisse** [comment]
Hi Heiko,

Harald has already merged this into master: https://github.com/hal/core/pull/26, hasn't he ?

Anyway, the BZ number in this PR is wrong, so I've done a new PR to superseed this one: [https://github.com/hal/core/pull/41]

**Comment by heiko-braun** [comment]
What's the correct BZ number?

**Comment by hpehl** [comment]
For the record: the correct BZ is [1191720](https://bugzilla.redhat.com/show_bug.cgi?id=1191720)

**Pull request closed without merging**